app/src/main/res/values/strings.xml

```diff
@@ -551,6 +551,8 @@
     <string name="format_error">Wrong Barcode/QR format</string>
     <string name="code_detected">We have detected the barcode</string>
     <string name="do_you_want_to">Do you want to add photos to this barcode?</string>
+    <string name="bottomNav_history">History</string>
+    <string name="search_bottomNav">Search</string>
```

Comment:
Can these two strings be standardised? They both flip where `bottomNav` is placed in the ID.